src/kosha/app/search.clj

```diff
@@ -3,13 +3,14 @@
             [kosha.app.util :as util]))
 
 
-(defn get-ragam
+(defn get-ragams
   "Retrieves 10 best matches for given ragam name."
-  [{:keys [params] :as request}]
-  (let [ragams (db-search/ragams (:q params) 10)]
+  [query]
+  (let [ragams (db-search/ragams query 10)]
```

Comment:
Do you think it's worthwhile now to add pagination to the search endpoint, and use this number as a maximum when pagination range is unspecified?